lib/opensource/Node-SMB/lib/ntlm.js

```diff
@@ -0,0 +1,236 @@
+/*
```

Comment:
Can we move this copyright to https://github.com/microsoft/typed-rest-client/blob/master/LICENSE

Reply:
We should also then be able to get rid of it from other files and shrink the library a little bit